src/main/scala/streamer/SpatialAddrGenUnit.scala

```diff
@@ -0,0 +1,112 @@
+package streamer
+
+import chisel3._
+import chisel3.util._
+
+// input and output decleration for unrolling adress generation unit
+// it's a submodule in side data mover
```

Comment:
```suggestion
// input and output declaration for unrolling adress generation unit
// it's a submodule of a data mover
```
Please provide clear docstrings instead!